Userland/Demos/Starfield/Starfield.cpp

```diff
@@ -47,6 +49,7 @@ class Starfield final : public GUI::Widget {
 
     Vector<Coordinate> m_stars;
     int m_sweep_plane = 2000;
+    int m_speed = 1;
```

Comment:
What happens if a negative speed is assigned? If that's not expected, this should be unsigned.